src/middlewares/AuthCheck.ts

```diff
@@ -0,0 +1,21 @@
+import UserService from "../services/UserService";
+import { IRequestHandler } from "../types/IRequestHandler";
+
+
+
+const AuthCheck = ( // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I prefer favoring classes over functions for this type of functionality. A base class will favor subclassing for reusability.

Reply:
This file was deleted